.mypy.ini

```diff
@@ -1,5 +1,5 @@
 [mypy]
-python_version = 3.11
+python_version = 3.13
```

Comment:
This `python_version` line can be removed entirely. Per [the docs](https://mypy.readthedocs.io/en/stable/config_file.html#confval-python_version), "the default is the version of the Python interpreter used to run mypy" and this should be enforced by `uv` now.